lib/lib-utils/src/main/java/org/sagebionetworks/util/json/translator/ByteArrayTranslator.java

```diff
@@ -0,0 +1,34 @@
+package org.sagebionetworks.util.json.translator;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+/**
+ * A translators between a byte[] and a base 64 String.
+ */
+public class ByteArrayTranslator implements Translator<byte[], String> {
+
+	/**
+	 * This can translate a byte[].
+	 */
+	@Override
+	public boolean canTranslate(Class<?> fieldType) {
+		return byte[].class.equals(fieldType);
+	}
+
+	@Override
+	public byte[] translateFromJSONToJava(Class<? extends byte[]> type, String jsonValue) {
+		return Base64.getDecoder().decode(jsonValue.getBytes(StandardCharsets.UTF_8));
```

Comment:
YOu probably want a constant for the decoder/encoder

Reply:
The method Base64.getDecoder() returns a private static decoder.  Same with the encoder.